akka-ssl-config/src/main/resources/reference.conf

```diff
@@ -0,0 +1,8 @@
+# Copyright (C) 2015 Typesafe Inc. <http://www.typesafe.com>
+
+# ssl configuration
+ssl-config {
+
+  logger = "com.typesafe.sslconfig.akka.util.AkkaLoggerBridge"
```

Comment:
This is correct, but needs documentation perhaps.
Generally speaking the "core ssl-config" wants to log things sometimes, and we need to provide it with the akka logger, thus this setting is for the "core ssl-config"